### PR TITLE
Bug 1264883 - use mozIntl to update the time information in sync tab.…

### PR DESCRIPTION
**Pull request by danhuang1202**
… r=zibi

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4c451ed7e8e7f6f2ab94393eba71b62e0e873b9d

File: tv_apps/browser/js/sync/tabList.js [hunk 1]
```diff
@@ -200,33 +200,31 @@
...
+        timeUnit = 'day';
+      } else if (hour) {
+        timeUnit = 'hour';
+      } else {
+        timeUnit = 'minute';
       }
 
+      let rtf = new mozIntl.RelativeTimeFormat(navigator.languages, {
+        unit: timeUnit
+      });
```
Comment (zbraniecki) [inline tv_apps/browser/js/sync/tabList.js]: Instead of this, you can use `unit: 'bestFit'` and RelativeTimeFormat will choose the appropriate unit.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c7708dd89a7740538c0b3a1e8dfcaa452dee65b0

File: tv_apps/browser/js/sync/tabList.js (hunk 1 above)
Comment (danhuang1202) [inline tv_apps/browser/js/sync/tabList.js]: Cool! It's really helpful!

**Pull request merged**